### PR TITLE
NEW Add dispatch-ci

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-dynamodb/issues/52

### CODE REVIEW

**Comment by GuySartorelli** [comment]
Can you please add some context to the description of the PR? I can't remember what this workflow is for or when we need to include it or really anything about it, so it's hard to review this PR.

**Comment by emteknetnz** [comment]
I adds this [workflow](https://github.com/silverstripe/gha-dispatch-ci) to modules - there's a description of what the module does in the readme

File: scripts/cms-any/dispatch-ci.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  if: (github.event_name == 'schedule' && github.repository_owner == '$account') || (github.event_name != 'schedule')
+  runs-on: ubuntu-latest
+  steps:
+    - name: Dispatch CI
+      uses: silverstripe/gha-dispatch-ci@v1
+EOT;
+
+if (check_file_exists('.github/workflows/ci.yml')) {
+    # If doing a mass update again in the future change this to write_file_even_if_exists()
+    # I would do this now, it's just I do not want to open 80+ PRs at present
```
Comment (GuySartorelli) [inline scripts/cms-any/dispatch-ci.php:37]: Please do this now and just don't run it immediately. Nobody is going to look at all of the scripts to check for TODO comments before running this tool.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: scripts/cms-any/dispatch-ci.php (hunk 1 above)
Comment (emteknetnz) [reply scripts/cms-any/dispatch-ci.php]: Done

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**